CONTRIBUTING.md

```diff
@@ -63,12 +63,13 @@ First, clone this repo. Then install dependencies (`npm i`). Now you're ready to
 
 If you want to use a particular dataset, you'll first need to place it in `/public/logs/nodes.json`. Otherwise, you can serve randomly-generated data.
 
-Kedro-Viz uses an environment variable to configure the data source. You can set it when starting up the dev server, e.g. `DATA=random npm start` will serve random procedurally-generated data (refreshed on each page-load). When random or mock data is enabled, certain other features like snapshot history are also enabled by default to help with browser testing. This is usually the most useful setting for local development.
+Kedro-Viz uses an environment variable to configure the data source. You can set it when starting up the dev server, e.g. `DATA=random npm start` will serve random procedurally-generated data (refreshed on each page-load). This is usually the most useful setting for local development.
```

Comment:
Would it be beneficial to have a script like "start:dev": "DATA=random npm start", I know i forget this when running and end up just using npm start - then staring at a blank pipeline lol

Reply:
Could do 🤷‍♂ though I'd need to add it several times. for each of the different options. I don't feel strongly either way.

Reply:
I'm going to leave this as-is for now, but I'll think about whether this could be done better. Right now I think the current setup (plus the URL query string method) work ok for changing the data input